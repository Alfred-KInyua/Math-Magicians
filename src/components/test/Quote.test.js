import React from 'react';
import renderer from 'react-test-renderer';
import Quotes from '../Quote';

it('Check if Quote renders correctly', () => {
  const tree = renderer.create(<Quotes />).toJSON();
  expect(tree).toMatchSnapshot();
});
