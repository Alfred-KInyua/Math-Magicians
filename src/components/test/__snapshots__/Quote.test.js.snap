// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Check if Quote renders correctly 1`] = `
<main>
  <div
    className="qoute"
  >
    <p
      className="pquote"
    >
      Mathematics is not about numbers, equations, computations or algorithms; it's about understanding. - William Paul Thurston
    </p>
  </div>
</main>
`;
